comm/src/main/scala/coop/rchain/kademlia/kademlia.scala

```diff
@@ -206,10 +205,10 @@ final case class PeerTable[A <: Peer](home: A,
       }
 
     distance(home.key, key) match {
-      case Some(index) => {
+      case Some(index) =>
         val entries = new mutable.ListBuffer[Entry]
 
-        for (i <- index to 8 * width - 1; if entries.size < k) {
+        for (i <- index until 8 * width; if entries.size < k) {
```

Comment:
Risky. I wouldnt touch this code till we have tests for it. Why did you changed that?

Reply:
This was an IDEA suggestion/optimization

Reply:
Will you take responsibility for the f** ups it might cause? :) We have 0 tests for this code, until we provide new Kademlia impl I dont want to touch this code